alts/scheduler/scheduling.py

```diff
@@ -49,6 +54,18 @@ def get_available_test_tasks(self) -> List[dict]:
             self.logger.exception('Cannot get available test tasks:')
         return response_as_json
 
+    def get_excluded_packages(self, distro: str) -> Optional[dict]:
+        if 'excluded_packages' not in self.__cached_config:
+            uri = f'{CONFIG.excluded_tests_url}'
+            try:
+                response = requests.get(uri, timeout=10)
+                response.raise_for_status()
+                self.__cached_config['excluded_packages'] = response.json()
+            except requests.RequestException:
+                return {}
+        pkgs = self.__cached_config.get('excluded_packages', {})
+        return pkgs.get(distro, {})
```

Comment:
please provide some examples of data that we expect to get for excluding packages in readme.md